Backend/WebApi.Weather/HourlyRequestBody.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace WebApi.Weather;
+
+public class HourlyRequestBody
+{
+    public string[]? Time { get; set; }
+    public int[] Weathercode { get; set; }
```

Comment:
Mamy w request podać weather code?

Reply:
Ja to rozumiem, w drugą strone, robimy zapytanie na te dwa elementy